doc/examples/moveit_cpp/src/moveit_cpp_tutorial.cpp

```diff
@@ -110,7 +112,8 @@ int main(int argc, char** argv)
     visual_tools.trigger();
 
     /* Uncomment if you want to execute the plan */
-    /* planning_components->execute(); // Execute the plan */
+    /* bool blocking = true; */
+    /* moveit_controller_manager::ExecutionStatus result = moveit_cpp_ptr->execute(plan_solution1.trajectory_, blocking, CONTROLLERS); */
```

Comment:
The commenting here is a bit tricky. I can't make an inline comment like this:

/* moveit_controller_manager::ExecutionStatus result = moveit_cpp_ptr->execute(plan_solution1.trajectory_, true /* blocking */, CONTROLLERS); */

And I can't make a static constant near the top of file or I get a compiler warning about an unused variable (since this is usually commented). This is the best compromise I could come up with.

We don't need to pass these `blocking` and `CONTROLLERS` args but I think it's good to start exposing new users to these options.